src/content_sao.cpp

```diff
@@ -91,6 +91,9 @@ class TestSAO : public ServerActiveObject
 	}
 
 	bool getCollisionBox(aabb3f *toset) const { return false; }
+
+	bool getSelectionBox(aabb3f *toset) const { return false; }
```

Comment:
okay, then just add a virtual keyword, this will permit to know directly this is a child derived function. I know it's not the case everywhere but we tend to add it for clarity (example: databases)